ibis/backends/datafusion/tests/test_client.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+import pyarrow as pa
+
+from .. import Backend
+from ..client import DataFusionTable  # noqa: E402
```

Comment:
I realize this is probably copied from another file, but relative imports in tests are a bad idea generally. Tests should import the library as a user would, and a user isn't going to be using imports relative to ibis when they are using ibis.